tests/framework/checks/skips.go

```diff
@@ -214,12 +214,16 @@ func SkipIfOpenShift4(message string) {
 	}
 }
 
-func SkipIfMigrationIsNotPossible() {
```

Comment:
Please refrain from introducing new skips or using the existing ones. Rather create a general label in `tests/decorators` that can be used to skip a test based on the test suite user`s needs and knowledge.

Reply:
updated, I don't know if it makes sense this way, please take a look